api/src/errors/web_error.rs

```diff
@@ -190,7 +190,7 @@ impl ConvertToWebError for SerdeError {
 impl ConvertToWebError for TariError {
     fn to_response(&self) -> HttpResponse {
         error!("Tari error: {}", self);
-        internal_error("Internal error: Problem with the Tari client")
+        internal_error("Internal error")
```

Comment:
Sorry, I should have specified that it needs to be a friendlier error that doesn't say Tari error. Maybe something like
```suggestion
        internal_error("There was an error during the ticket transfer process")
```